packages/mdc-radio/index.js

```diff
@@ -84,15 +102,24 @@ export class MDCRadio extends MDCComponent {
     return new MDCRipple(this.root_, foundation);
   }
 
+  /**
+   * Returns the state of the native control element, or undefined if the native control element is not present.
+   * @return {SelectionControlState|undefined}
```

Comment:
`{SelectionControlState|undefined}` -> `{?SelectionControlState}`

`querySelector()` returns `null`, not `undefined`.